admin.py

```diff
@@ -1,25 +1,32 @@
 def add_drink(drinks, title, price):
-    drinks[title] = price
+    if title not in drinks:
```

Comment:
Maybe you should reply to the user that the drink was not found?

Reply:
Why answer that the drink was not found? This is the function of adding a drink, and if the drink I am trying to add is not in the dictionary, I will add it